bittide/src/Bittide/DoubleBufferedRam.hs

```diff
@@ -386,6 +387,31 @@ blockRamByteAddressableU readAddr newEntry byteSelect =
   rstFunc = clashCompileError "blockRamByteAddressableU: reset function undefined"
 
 data RegisterWritePriority = CircuitPriority | WishbonePriority
+ deriving Eq
+
+-- | Register with additional wishbone interface, this component has a configurable
+-- priority that determines which value gets stored in the register during a write conflict.
+-- With 'CircuitPriority', the incoming value in the fourth argument gets stored on a
+-- collision and the wishbone bus gets acknowledged, but the value is silently ignored.
+-- With 'WishbonePriority', the incoming wishbone write gets accepted and the value in the
+-- fourth argument gets ignored.
```

Comment:
Nitpick, there's no "fourth argument", this function only has two arguments :nerd_face: 

I guess you mean of the `Circuit` inner function?